Formula/ghc@9.rb

```diff
@@ -44,13 +51,15 @@ def install
     resource("binary").stage do
       binary = buildpath/"binary"
 
-      system "./configure", "--prefix=#{binary}"
+      system "./configure", "--prefix=#{binary}", "--with-gmp-includes=#{Formula["gmp"].opt_include}",
+             "--with-gmp-libraries=#{Formula["gmp"].opt_lib}"
       ENV.deparallelize { system "make", "install" }
 
       ENV.prepend_path "PATH", binary/"bin"
     end
 
-    system "./configure", "--prefix=#{prefix}"
+    system "./boot"
```

Comment:
Why's this + autotools needed now?

Reply:
It's because the 9.2.3 tarball doesn't have a `configure` script, so it needs to be generated.  I'm not sure why this has changed but it may be in anticipation of the eventual transition to building with Hadrian.

Reply:
https://gitlab.haskell.org/ghc/ghc/-/issues/21673 reported to GHC, and labeled as a bug